native-hello-module/README.md

```diff
@@ -1,54 +1,35 @@
-# GraalVM Demos: Building a HelloWorld Java Module into a Native Executable
-This repository contains the code for a demo application for [GraalVM](http://graalvm.org).
+# Build a Java Module into a Native Executable
 
-## Prerequisites
-* [GraalVM](http://graalvm.org)
-* [Native Image](https://www.graalvm.org/docs/reference-manual/native-image/)
-* Maven
+GraalVM Native Image supports the Java Platform Module System, introduced in Java 9, which means you can convert a modularized Java application into a native executable.
 
-## Preparation
+The `native-image` tool accepts the module-related arguments like `--module (-m)`, `--module-path (-p)`, `--add-opens`, `--add-exports` (same as for the java launcher). When such a module-related argument is used, the native-image tool itself is used as a module too.
```

Comment:
The native-image tool isn't wrapped in code tags here:

... the native-image tool itself is used as a module too